src/psyclone/psyir/nodes/extract_node.py

```diff
@@ -97,15 +94,7 @@ def extract_body(self):
         :raises InternalError: if this node does not have a single Schedule as\
```

Comment:
No, this check is still there ... in the base class. I am not sure what our policy is re exceptions raised in a base class - imho it will be near impossible to keep them all up-to-date, but I think it makes sense to leave the documentation we have here.

Reply:
Our (probably unwritten) rule is to only document the exceptions raised by the current method. Those raised by the base class are documented in the base class.

Reply:
I've made this into a written rule in the coding style, and remove the comment.

Reply:
Thanks.